azure-sdk-testutils/devtools_testutils/mgmt_testcase.py

```diff
@@ -157,12 +157,16 @@ def create_mgmt_client(self, client_class, **kwargs):
             self.create_basic_client(
                 client_class,
                 subscription_id=None,
+                resource_group_name=None,
```

Comment:
Please remove that. In general, I don't see why you need any changes at all in this file.